### PR TITLE
Update documentation for Wagtail 2.3

### PR DESCRIPTION
**Pull request by cwdavies**
Change wagtail links for v1.x to stable in documents and code comments

## Checklist

- [X] PR has an informative and human-readable title
- [X] Changes are limited to a single goal (no scope creep)
- [X] Code can be automatically merged (no conflicts)
- [X] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [X] Project documentation has been updated
- [X] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by chosak** [comment]
Can we also update these links in code comments?

```sh
$ git grep "v1.13.4" -- "*.py"
cfgov/scripts/initial_data.py:        # https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail/wagtailcore/migrations/0002_initial_data.py#L33
cfgov/v1/tests/test_wagtail_hooks.py:    # Borrowed from https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail
cfgov/v1/wagtail_hooks.py:    https://docs.wagtail.io/en/v1.13.4/reference/hooks.html#register-admin-menu-item
```

**Comment by Scotchester** [comment]
I recognize that you were doing as the acceptance criterion requested, @cwdavies, but I wonder if we should be more forward looking to avoid having to update these links frequently. Are we optimistic enough about getting to 2.8 quickly and keeping up with new releases to use the `stable` docs, rather than a specific version number?

Curious to hear from @higs4281 @willbarton @chosak @niqjohnson on this.

**Comment by cwdavies** [comment]
I agree with comments from both @Scotchester and @chosak and have changed all wagtail document versions from `v1.x` and `v2.3` to be `stable`. In the code comments I have modified the directory path from `\wagtail\wagtailcore\` to `\wagtail\core\` and used `master` branch.
I am curious to hear from @higs4281 @willbarton @chosak @niqjohnson on this.
After there is a release that is deployed to production that uses Wagtail 2.3, and when we clean up the Python code to remove support for Wagtail 1.x, the code comments should be changed.

**Comment by higs4281** [comment]
team stable

File: cfgov/v1/tests/test_wagtail_hooks.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def test_tag_param_returns_resources_tagged_that_way(self):
 
 @unittest.skipIf(wagtail.VERSION >= (2, 0), "No need to test in Wagtail 2+")
 class TestWhitelistOverride(SimpleTestCase):
-    # Borrowed from https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail
-    # /wagtailcore/tests/test_dbwhitelister.py
+    # Borrowed from https://github.com/wagtail/wagtail/blob/master/wagtail
+    # /core/tests/test_whitelist.py
 
     def test_whitelist_hooks(self):
         """Test that DbWhitelister does not strip new elements and attributes.
```
Comment (higs4281) [inline cfgov/v1/tests/test_wagtail_hooks.py]:
```suggestion
        """Test that Whitelister does not strip new elements and attributes.
```

Minor note, but the class dropped the "Db" prefix.

**Pull request merged**